components/carousel/overview.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Overview
+page_title: Overview | Carousel for Blazor
+description: Overview of the Carousel for Blazor.
+slug: carousel-overview
+tags: telerik,blazor,carousel,overview
+published: True
+position: 0
+---
+
+# Carousel Overview
+
+The <a href = "https://www.telerik.com/blazor-ui/carousel" target="_blank">Carousel for Blazor</a> is an interactive component that allows users to browse a collection of content items (pages) one at a time. The Carousel template supports random web content, although it is most often used to display images.
+
+#### In this article:
+   * [Basics](#basics)
+   * [Example](#example)
+   * [Features](#features)
+
+## Basics
+
+To use a Telerik Carousel for Blazor:
+
+1. Add the `TelerikCarousel` tag.
+1. Set the `Data` attribute to the variable that holds the component data.
+1. Add a nested [`<Template>`]({%slug carousel-template%}) tag that will define the HTML content to display.
+1. Set `Width` and `Height` attributes of the Carousel, according to the content to display.
+
+## Example
+
+Here is a simple Carousel that has 5 pages with some styled text.
+
+````CSHTML
```

Comment:
Consider making an example with a model as it will feel more real-life to people.